addons/event/views/event_views.xml

```diff
@@ -620,6 +629,8 @@
                 <search string="Event Registration">
                     <field name="id" string="Registration ID"/>
                     <field name="name" string="Participant" filter_domain="['|', '|', ('name', 'ilike', self), ('email', 'ilike', self), ('origin', 'ilike', self)]"/>
+                    <filter string="Ongoing Events" name="ongoing_events" domain="[('event_id.is_ongoing', '=', True)]"/>
```

Comment:
I would name it filter_is_ongoing. This convention is not enforced but we try to name filter with name_<field_name>, easier to find when searching. To propagate to enterprise PR if you change :) .